go/vt/vttablet/tabletserver/planbuilder/dml.go

```diff
@@ -470,12 +470,6 @@ func analyzeInsertMessage(ins *sqlparser.Insert, plan *Plan, table *schema.Table
 		return nil, fmt.Errorf("%s must be specified for message insert", col.String())
 	}
 
-	col = sqlparser.NewColIdent("message")
-	num = findCol(col, ins.Columns)
-	if num < 0 {
-		return nil, fmt.Errorf("%s must be specified for message insert", col.String())
-	}
-
```

Comment:
The intent of this check is gone, does it still make sense to check for some payload column to be checked?

Also, I guess we now support having no payload columns too, that may be a good feature.